### DISCUSSION TITLE
Une autre formulation sur la page Stats?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1783] [Idées]
Sur la page Stats, une utilisatrice pensait que la formule "X % ont réalisé la mesure « qualité des produits »" signifiait "X % des cantines ont mesuré la qualité de leur produits", donc "X % des cantines ont renseigné des chiffres d'approvisionnement". Alors que le chiffre désigne en réalité le pourcentage de cantines qui ont atteint les objectifs EGAlim. Je trouve aussi que la formule peut prêter à confusion. Je suppose que "X % ont atteint les objectifs EGAlim" serait plus claire.

@gauthierandre, @Greta712, Pauline, avez-vous eu des commentaires similaires?
@valeriemerle, @JenniferStephan, qu'en pensez-vous?

<img width="1132" alt="Capture d’écran 2022-08-16 à 15 40 26" src="https://user-images.githubusercontent.com/93709421/184896449-89e1216d-05ca-40d3-8b33-938579a58f27.png">

**Comment by alemangui** [comment]
Tu as raison, je pense que le mot « mesure » porte à confusion. Je propose "x% ont réussi l'objectif d'approvisionnement EGAlim" :thinking:

**Reply by alemangui** [reply]
Carte ajoutée dans le projet :+1:

**Reply by ghost** [reply]
Thanks!
